src/contentsharecontroller/ContentShareControllerFacade.ts

```diff
@@ -14,6 +14,11 @@ export default interface ContentShareControllerFacade {
    */
   startContentShareFromScreenCapture(sourceId?: string): Promise<void>;
 
+  /**
+   * Set the max frame rate for screen capture. By default, the max frame rate is 15.
+   */
+  setScreenCaptureMaxFrameRate(frameRate: number): void;
```

Comment:
Instead, could we add a parameter to `startContentShareFromScreenCapture` like this
`startContentShareFromScreenCapture(sourceId?: string | null, frameRate?: number): Promise<void>;` ?